src/index.test.js

```diff
@@ -240,7 +231,8 @@ test('reads text from fs if filePath is provided but not text', () => {
   } catch (e) {
     // ignore
   }
-  expect(fsMock.readFileSync).toHaveBeenCalledTimes(1)
+  // Two hits to .eslintignore
```

Comment:
And here's why it reads it two times: https://github.com/eslint/eslint/blob/master/lib/ignored-paths.js#L159-L160